playbook/app/pb_kits/playbook/pb_stat_change/_stat_change.tsx

```diff
@@ -0,0 +1,68 @@
+/* @flow */
```

Comment:
Once it's been converted to TS, you can remove the @flow from the top of the file Israel!